### PR TITLE
BLP-3

### PR DESCRIPTION
**Pull request by ChiuMungZitAlexander**
add webpack configuration

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                    { loader: "style-loader" },
+                    { loader: "css-loader" },
+                    { loader: "sass-loader" }
+                ]
+            }
+        ]
+    },
+    devServer: {
+        contentBase: './build',
+        // colors: true,
```
Comment (beltekylevi) [inline webpack.config.js]: Do not leave comments in the code.

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        historyApiFallback: true,
+        inline: true,
+        port: 3000
+        // process: true
```
Comment (beltekylevi) [inline webpack.config.js]: Do not leave comments in the code.

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    output: {
+        filename: 'bundle.js',
+        path: __dirname + 'build'
+    },
+    module: {
+        loaders: [
+            {
+                test: /\.(js|jsx)$/,
+                exclude: /node_modules/,
+                loader: 'babel-loader'
```
Comment (beltekylevi) [inline webpack.config.js]: Add the necessary presets here

**Review by beltekylevi — CHANGES_REQUESTED** [review]
Use 2 spaces for indentation.
Save the necessary development dependencies like `sass-loader`, `css-loader`...

**Review by beltekylevi — CHANGES_REQUESTED** [review]
Initialize an npm project (with `npm init`) and add the necessary packages as dependencies (webpack, loaders, etc)

**Comment by ChiuMungZitAlexander** [comment]
add package.json

**Review by ChiuMungZitAlexander — COMMENTED** [review]
add package.json

**Review by ChiuMungZitAlexander — COMMENTED** [review]
add package.json

File: package.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "epam-jsa-black-pearl",
+  "version": "1.0.0",
+  "description": "Black Pearl workspace -- Chris Alex Alfred",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack"
```
Comment (beltekylevi) [inline package.json]: Creating a task to run webpack is not part of the BLP-3, please remove

**Review by beltekylevi — CHANGES_REQUESTED** [review]

File: package.json [hunk 5]
```diff
@@ -5,13 +5,13 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "webpack"
+    "lint": "eslint src"
```
Comment (beltekylevi) [inline package.json]: BLP-3 has nothing to do with the linter, remove this line please

**Review by beltekylevi — CHANGES_REQUESTED** [review]

**Pull request merged**